File: Assets/Scripts/Network/SimplifiedClass/Player/Player.cs

namespace SerializableClass
{

    [System.Serializable]
    public class Player
    {
        public string id;
        public string username;
        public Characteristic characteristic;
        public MyVector3 positionInWorld;
        public MyVector2 positionArrayMain;
        public MyVector2 positionArrayFight;

        public Player()
        {
            positionInWorld = new MyVector3();
            positionArrayMain = new MyVector2();
            positionArrayFight = new MyVector2();
        }
    }

    [System.Serializable]
    public class Characteristic
    {
        public string name;
        public int baseHealthPoints;
        public int baseActionPoints;
        public int baseMovementPoints;
        public string[] spells;
        public Spell[] myspells;
}
}

File: Server/Characters/Player/Player.js
var Vector3 = require('../../utils/MyVector3')
var Vector2 = require('../../utils/MyVector2')

var Character = require('../Character');

module.exports = class Player extends Character {
    constructor(username, baseCharacteristic, spells, isAI = false, isTrainable = false) {

        super(baseCharacteristic, spells, isAI, isTrainable);
        this.username = username;

        //Pos
        this.positionInWorld = new Vector3();
        this.positionArrayMain = new Vector2();
    }

}
